merlin/models/tf/distributed/backend.py

```diff
@@ -0,0 +1,27 @@
+import tensorflow as tf
+
+from merlin.core.dispatch import HAS_GPU
+
+hvd = None
+multi_gpu = False
+
+if HAS_GPU:
+    try:
+        import horovod.tensorflow.keras as hvd
+
+        multi_gpu = True
```

Comment:
It seems that we will do always multi_gpu training, is that correct? I can see a case, where we have multiple GPUs on 1 machine but we want to do single GPU training. E.g. I have 4x GPUs, but I want to run 4x independent experiments in parallel

Reply:
In this draft, I was thinking users could just set the global variable to False, e.g., `distributed.multi_gpu = False`, if they wanted to do single GPU training. But now I think we should drop this entirely. Users would be running the script with `horovodrun -np N script.py`, so if they want to do multi-gpu training, they can control the number of processes with `-np`. Or, if they simply do not use `horovodrun` or `mpirun`, it would just use a single GPU and `hvd.size()` would be `1`. Some of my thoughts here: https://github.com/NVIDIA-Merlin/Merlin/issues/651#issuecomment-1272785284